openpype/hosts/maya/api/lib_renderproducts.py

```diff
@@ -625,12 +634,28 @@ def _get_aov_render_products(self, aov, cameras=None):
                             aov=aov_name,
                             driver=ai_driver,
                             ext=ext,
-                            camera=camera
+                            camera=camera,
+                            colorspace=colorspace
                         )
                         products.append(product)
 
         return products
 
+    def _get_colorspace(self, attribute):
+        """Resolve colorspace from Arnold settings."""
+
+        def _view_transform():
+            preferences = lib.get_color_management_preferences()
+            return preferences["view_transform"]
+
+        resolved_values = {
+            "Raw": lambda: "Raw",
```

Comment:
As far as I know `Raw` is not a valid colorspace. It might be for the default config but there's no specification requirement that a `Raw` key exists. I believe when an image is used 'raw' to Arnold what it means is that it's actually in Arnold rendering space. That is the `cmds.colorManagementPrefs(query=True, renderingSpaceName=True)`.

Reply:
Good point!